skimage/transform/_warps.py

```diff
@@ -18,14 +17,27 @@
 )
 
 
+def _multichannel_default(multichannel, ndim):
+    if multichannel is not None:
+        return multichannel
+    else:
+        warn('The default multichannel argument (None) is deprecated. '
+             'Specifiy True or False explicitly.')
```

Comment:
actually `_multichannel_default` is fine as is (aside from the warnings text).  It is only used by `rescale` and the various pyramid functions which all previously assumed 2D+channels if the input was 3D and would error on inputs > 3D.